packages/core/core/src/ParcelConfig.schema.js

```diff
@@ -0,0 +1,117 @@
+// @flow strict-local
+import type {FilePath} from '@parcel/types';
+import type {SchemaEntity} from '@parcel/utils';
+import {validatePackageName} from './loadParcelConfig';
+
+const validatePluginName = (
+  pluginType: string,
+  key: string,
+  relativePath: FilePath
+) => {
+  return (val: string) => {
+    // Skip plugin spread...
+    if (val === '...') return;
+
+    try {
+      validatePackageName(val, pluginType, key, relativePath);
```

Comment:
Nice! We could probably shorten the error message a bit now that we have the diagnostic. `Official parcel ${pluginType} packages must be named according to "@parcel/${pluginType}-{name}"` is probably enough: the actual value and location of the config file is visible in the codeframe.

Reply:
updated it, thought it was still used elsewhere but apparently I was mistaken